src/serialize_sequence.cc

```diff
@@ -182,4 +186,22 @@ void store_sequence(const std::filesystem::path& folder, const Sequence& seq)
         store_step(seq_path / extract_filename_step(++idx, max_digits, step), step);
 }
 
+
+void remove_sequence(const std::filesystem::path& seq_path)
+{
+    try
+    {
```

Comment:
I guess we need some safety check here against malicious users passing in "../../home" or something like that.

Reply:
I can filter for leading '/' to prevent an absolute path

Reply:
See the last commit. I handle it now in the SequenceManager where I have the member variable "path_". I concat path_ with seq_path and check for existence. This guarantees a relative path with the sequence folder as base